### PR TITLE
a11y: change button to icon in FieldLabel.tsx

### LINKED ISSUE #2115: [Accessibility][Screen reader - Property Pane_Ask a question (Bot Asks)]: Incorrect role is announced for ? as button in property pane.
**User Experience** 
User who rely on screen reader will get confused if incorrect role is announced by the screen reader.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen.
5. Now, navigate to '?' just beside of 'Prompt.
6. Observe the issue.

**Actual:** 
On getting the focus, narrator announces role as button along with the descriptive information.

**Expected:**
As it has no functionality, the narrator shouldn't announce the control as button.

**Note:**
Same issue is observed for all the Property Pane which have "?".

**MAS Reference:** [MAS4.2.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment:**
[MAS4.1.2_incorrect role is defined.zip](https://github.com/microsoft/BotFramework-Composer/files/4261888/MAS4.1.2_incorrect.role.is.defined.zip)
MAS4 1 2_Incorrect role is defined for question mark button

### PR DESCRIPTION
**Pull request by beyackle**
## Description

Replaces the icon button in FieldLabel with an ordinary Icon (and makes sure it's still in screen-reader tabbable order).

## Task Item

Closes #2115

## Screenshots

Also just a minor adjustment so the focus rectangle looks better.

image

### CODE REVIEW

**Comment by cwhitten** [comment]
@beyackle can we bump the focus indication over a few pixels to not overlap on top of the label?

**Comment by beyackle** [comment]
@cwhitten Will do.

**Comment by beyackle** [comment]
Updated spacing.

image

File: Composer/packages/extensions/adaptive-form/src/components/FieldLabel.tsx [hunk 1]
```diff
@@ -32,7 +32,9 @@ const DescriptionCallout: React.FC<DescriptionCalloutProps> = function Descripti
         styles: { root: { width: '288px', padding: '17px 28px' } },
         onRenderContent: () => (
           <div>
-            <h3 style={{ fontSize: '20px', margin: '0', marginBottom: '10px' }}>{title}</h3>
+            <h3 style={{ fontSize: '20px', margin: '0', marginBottom: '10px' }} aria-label={title + '.'}>
```
Comment (a-b-r-o-w-n) [inline Composer/packages/extensions/adaptive-form/src/components/FieldLabel.tsx:35]: Can you explain why adding a `.` for SR is important?

Comment (beyackle) [reply Composer/packages/extensions/adaptive-form/src/components/FieldLabel.tsx:35]: Without it, the SR runs the header and explanation together without a pause.

**Pull request merged**